afinn/data/AFINN-no-165.txt

```diff
@@ -0,0 +1,3214 @@
+infantilized	-2
+infested	-2
+infesting	-2
+infests	-2
+infract	-2
+infracted	-2
+infracting	-2
+infracts	-2
+innovates	1
+insolvent	-2
+intelligent	2
+intransigency	-2
+irreversible	-1
+jackass	-4
+jackasses	-4
+jesus	1
+jolly	2
+jovial	2
+jubilant	3
+kickbacks	-3
+kidnapping	-2
+lackadaisical	-2
+lag	-1
+lagged	-2
+lagging	-2
+lame	-2
+lapse	-1
+lawl	3
+likable	2
+likers	2
+lmao	4
+lmfao	4
+loathes	-3
+lobby	-2
+lobbyist	-2
+lol	3
+lolol	4
+lololol	4
+lolololol	4
+lool	3
+loool	3
+looool	3
+lovelies	3
+lugubrious	-2
+lurking	-1
+madly	-3
+maladaption	-2
+maldevelopment	-2
+marvels	3
+meditative	1
+memoriam	-2
+messed	-2
+mirthfully	3
+misbehaved	-2
+misbehaves	-2
+misbranding	-3
+miscast	-2
+mischiefs	-1
+misclassified	-2
+misclassifies	-2
+misconducted	-2
+misconducting	-2
+misconducts	-2
+misgiving	-2
+misleaded	-3
+misplacing	-2
+misread	-1
+misreport	-2
+misreported	-2
+misreports	-2
+moaning	-2
+mongering	-2
+monopolizes	-2
+monotone	-1
+mope	-1
+mourns	-2
+mumpish	-2
+murdering	-3
+n00b	-2
+niggas	-5
+nigger	-5
+offline	-1
+oops	-2
+optionless	-2
+ostracize	-2
+ostracizes	-2
+overselling	-2
+oversimplifies	-2
+overstatements	-2
+oxymoron	-1
+parley	-1
+pesky	-2
+pique	-2
+pissing	-3
+plodding	-2
+prblm	-2
+prblms	-2
+problem	-2
+propaganda	-2
+prosecutes	-1
+protest	-2
+pushy	-1
+rageful	-2
+ranter	-3
+rants	-3
+raptures	2
+relishing	2
+reprimanding	-2
+robed	-2
+robing	-2
+robust	2
+rofl	4
+roflcopter	4
+romantical	2
+rose	1
+rotflmfao	4
+rotflol	4
+ruin	-2
+sappy	-1
+savange	-2
+savanges	-2
+sexistic	-2
+sexy	3
+shitty	-3
+shoody	-2
+slam	-2
+slashing	-2
+slave	-3
+slickest	2
+slip	-1
+slumping	-1
+smart	1
+smog	-2
+snub	-2
+snubs	-2
+solidifying	2
+spam	-2
+spammer	-3
+spamming	-2
+spirited	2
+squanders	-2
+squelched	-1
+stall	-2
+stinked	-2
+stressor	-2
+stressors	-2
+stunned	-2
+sueing	-2
+suicidal	-2
+sulking	-2
+sulky	-2
+super	3
+supporter	1
+supreme	4
+talent	2
+terrorist	-2
+tolerant	2
+tout	-2
+touting	-2
+touts	-2
+transgresses	-2
+transgressing	-2
+tremor	-2
+troll	-2
+twat	-5
+tyrannic	-3
+ugh	-2
+unappeasable	-2
+unavailing	-2
+uncredited	-1
+unfunny	-2
+ungenerous	-2
+unimpressed	-2
+uninvolving	-2
+unlovable	-2
+unparliamentary	-2
+unreleting	-2
+unresearched	-2
+upsetting	-2
+uptight	-2
+vapid	-2
+vexing	-2
+victimization	-3
+victimizes	-3
+victimizing	-3
+victor	3
+vindicated	2
+vindicates	2
+vitamin	1
+vitriolic	-3
+vomits	-3
+walkouts	-2
+wanker	-3
+winwin	3
+wits	2
+woebegone	-2
+woo	3
+woohoo	3
+wooo	4
+woow	4
+wow	4
+wowww	4
+wrenching	-2
+wtf	-4
+wtff	-4
+wtfff	-4
+xo	3
+xoxo	3
+xoxoxo	4
+xoxoxoxo	4
+yeees	2
+yucky	-2
+søtt	2
+forbløffer	1
+sjarmerte	3
+ikke livsfarlig	2
+fantastiske	4
+forsvarte	1
+hyller	1
+verdensmester	2
+verdensmesteren	2
+sol	1
+sommer	1
+sommerferie	1
+ferie	1
+reddes	2
+brakseier	2
+drømmetur	2
+årets	1
+knockoutseier	1
+yndig	2
+perfekte	3
+magisk	2
+juleferiring	1
+priskrig	1
+største	1
+idealist	1
+på beina igjen	2
+gleder	3
+kåret	2
+drømmejobb	2
+tjener	1
+kommer til oslo	2
```

Comment:
Does this phrase and the 3 below make sense to be given a sentiment value?

Reply:
I can remove these as they are maybe just relevant to my usecase. 

As I deal mainly with news and if they write that someone is coming to the city it is generally someone famous and the sentence therefore has a positive sentiment.